### PR TITLE
[MRG] Bug fix with plot_raw

### PR DESCRIPTION
**Pull request by jaeilepp**
Noticed a bug. If you try ``mne.viz.plot_raw(raw)``, it failed because ``n_channels=None``.

### CODE REVIEW

**Comment by Eric89GXL** [comment]
I guess we had been doing `raw.plot(...)` in all instances, whoops. Can you change one of the `raw.plot` instances in tests to do `mne.viz.plot_raw(...)` to help catch any future issues?

File: mne/viz/tests/test_raw.py [hunk 1]
```diff
@@ -85,7 +86,7 @@ def test_plot_raw():
         # Color setting
         assert_raises(KeyError, raw.plot, event_color={0: 'r'})
         assert_raises(TypeError, raw.plot, event_color={'foo': 'r'})
-        fig = raw.plot(events=events, event_color={-1: 'r', 998: 'b'})
+        fig = plot_raw(raw, events=events, event_color={-1: 'r', 998: 'b'})
```
Comment (agramfort) [inline mne/viz/tests/test_raw.py]: why this change?

Comment (jaeilepp) [inline mne/viz/tests/test_raw.py]: Isn't this what @Eric89GXL requested?

Comment (Eric89GXL) [inline mne/viz/tests/test_raw.py]: It's called TDD @agramfort :)

Comment (agramfort) [inline mne/viz/tests/test_raw.py]: Trick Daddy Dollar ? ;)

**Comment by agramfort** [comment]
travis broken for weird reasons. Any idea @Eric89GXL ?

**Comment by Eric89GXL** [comment]
Same failures as in #2701, I'm working on fixing them there

**Comment by agramfort** [comment]
ok I'll let you cleanup your mess ;)

**Comment by Eric89GXL** [comment]
Oh I didn't cause them, they cropped up probably due to `conda` updates

**Comment by agramfort** [comment]
ok

**Comment by Eric89GXL** [comment]
Thanks @jaeilepp
